src/main/java/alexiil/mc/mod/load/Tips.java

```diff
@@ -23,6 +25,19 @@ public class Tips {
     }
 
     public static void load() {
+        File options = new File("options.txt");
+        try (BufferedReader reader = new BufferedReader(new FileReader(options))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(":");
+                if (parts[0].equals("lang")) {
```

Comment:
```suggestion
                if (parts.length > 1 && parts[0].equals("lang")) {
```
Just in case. (I know a valid `options.txt` won't have any, but otherwise this would result in a full crash)